src/public/constants.ts

```diff
@@ -4,6 +4,7 @@ export enum HostClientType {
   android = 'android',
   ios = 'ios',
   rigel = 'rigel',
+  surfaceHub2 = 'surfaceHub2',
```

Comment:
as discussed please rename it to surfaceHub

Reply:
changed.